src/main/java/sg/ncl/testbed_interface/User2.java

```diff
@@ -11,6 +16,7 @@
 public class User2 implements Serializable {
 
     private String id;
+    private ZonedDateTime applicationDate;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this value-based field transient so it is not included in the serialization of this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3437)